config/settings.yml.example

```diff
@@ -30,7 +30,9 @@
 # Uncomment and modify if you want to change the default pid file '/var/run/foreman-proxy/foreman-proxy.pid'
 #:daemon_pid: /var/run/foreman-proxy/foreman-proxy.pid
 
-# HTTP ports configuration
+# host and ports configuration
+# host to bind ports to (default: 0.0.0.0 for production, localhost for development)
```

Comment:
also, we set environment to production, so it always defaults to 0.0.0.0. As I suggested in my other comment, I think it would be better to define the default value in global settings, and override it in the config file, as with other settings.